### ISSUE TITLE
DataTable: Reorder (Drag and Drop) not working on touch device.

### ISSUE THREAD

**Issue by Boldonglen** [primefaces/primevue#5805]
### Describe the bug

When trying to reorder the DataTable on a touch device (i.e. mobile or tablet) the drag and drop functionality does not work. I have found a few other GitHub issues describing the issue with no fix.

To replicate this issue you just need to use a mobile device (or your browsers mobile emulator) and try to drag and drop a reorderable column or row inside a DataTable. This can even be seen in the examples within the official documentation. https://primevue.org/datatable/#reorder

This issue has been raised multiple times with no resolution:
https://github.com/primefaces/primevue/issues/3700
https://github.com/primefaces/primevue/issues/1543

### Reproducer

https://primevue.org/datatable/#reorder

### PrimeVue version

3.52.0

### Vue version

3.x

### Language

ALL

### Build / Runtime

Vite

### Steps to reproduce the behavior

Go to https://primevue.org/datatable/#reorder on a touch device (or in dev tools mobile emulator).
Touch one of the draggable parts of the Data Table
Try and drag the Column or Row
The Column or Row is not draggable

### Expected behavior

When the orderable item is pressed and held the item should be dragggable.

**Comment by 1Map** [comment]
@Boldonglen Yes, also find this not working on my side.  Even on the column resize:

https://primevue.org/datatable/#column_resize
